torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -827,7 +826,17 @@ class IrParser {
 
   bool registerTensor(const JitValue* val) {
     CgValue cg_val;
-    if (auto tensor_type = val->type()->cast<TensorType>()) {
+    // Don't register if we don't support the type
```

Comment:
If we try to lift the error the caller of this function, can we update the error message here to point to the failing value?
https://github.com/csarofeen/pytorch/blob/7b54fa7086787b34cb2ca655ba773f81fd68d5c0/torch/csrc/jit/codegen/cuda/parser.cpp#L74-L76

Would be nice that we dump the type. 
```
Failure when register value in codegen with type: ",
val.type().str()
```